hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java

```diff
@@ -230,12 +230,11 @@ public String getNamespace() {
       datanodeDetails.setRevision(
           HddsVersionInfo.HDDS_VERSION_INFO.getRevision());
       TracingUtil.initTracing(
-          "HddsDatanodeService." + datanodeDetails.getUuidString()
-              .substring(0, 8), conf);
+          "HddsDatanodeService." + datanodeDetails.getID(), conf);
```

Comment:
`substring` is missing, but this is only for distributed tracing, so I think it's OK.